site/package.json

```diff
@@ -14,6 +14,7 @@
     "escape-string-regexp": "2.0.0",
     "gatsby": "2.8.3",
     "gatsby-plugin-emotion": "4.0.7",
+    "gatsby-plugin-matomo": "^0.7.2",
```

Comment:
We're not using ^, we're using fixed versions

Reply:
done